src/Comms.cpp

```diff
@@ -22,9 +41,59 @@ namespace Comms {
     }
 
     void emitPacket(Packet *packet) {
-        Serial.print(packet->data[0]);
-        Serial.print(packet->data[1]);
-        Serial.print(packet->data[2]);
-        Serial.println(packet->data[3]);
+        //calculate and append checksum to struct
+        uint16_t checksum = computePacketChecksum(packet);
+        uint8_t *ptr = (uint8_t *)&checksum;
+        packet->checksum[0] = ptr[0];
+        packet->checksum[1] = ptr[1];
+
+        //Send over serial
+        Serial.write(packet->id);
+        Serial.write(packet->len);
+        Serial.write(packet->checksum, 2);
+        Serial.write(packet->data, packet->len);
+        Serial.write('\n');
+
+        //Send over ethernet
+        Udp.beginPacket(Udp.remoteIP(), Udp.remotePort());
```

Comment:
Here, the port can just be 42069 like it is above. The IP should actually be `10.0.0.69` and `10.0.0.70`. So this code needs to actually run twice to send to both IP addresses.